src/containers/App.js

```diff
@@ -1,9 +1,10 @@
 import React, { useState } from 'react';
 import './App.css';
-import Card from './Card';
-import Header from './Header';
+import CardList from '../components/Cardlist/Cardlist';
```

Comment:
Have you forgotten about correct naming/reference practices for React components? file should better be named `index.js`.